src/core/ui/components/drop-in/drop-down.tsx

```diff
@@ -0,0 +1,414 @@
+            </div>
+          </>
+        )}
+      </div>
+      {isOpen && (
+        <Content>
+          <Section style={{ borderTop: 'none' }}>
+            {/*<Name>{user.name}</Name>*/}
+            {hashDisplay ? (
+              <Row
+                style={{
+                  ...primaryIdStyle,
+                }}
+              >
+                {hashDisplay}
+                <Icon
+                  style={{ marginLeft: 8, cursor: 'pointer' }}
+                  onClick={() => {
+                    navigator.clipboard.writeText(hash);
+                  }}
+                >
+                  {copyIcon}
+                </Icon>
+              </Row>
+            ) : (
+              <Row
+                style={{
+                  paddingTop: 8,
+                  color: '#2F5FFC',
+                }}
+                onClick={() => {
+                  connect(false);
+                }}
+              >
+                <Icon>{plusIcon}</Icon>
+                Connect web3 wallet
+              </Row>
+            )}
+          </Section>
+          {/*<Section>*/}
+          {/*  <Row>*/}
+          {/*    <Icon>{twitterIcon}</Icon>*/}
+          {/*    @0xhaileym*/}
+          {/*  </Row>*/}
+          {/*  <Row*/}
+          {/*    style={{*/}
+          {/*      paddingTop: 8,*/}
+          {/*    }}*/}
+          {/*  >*/}
+          {/*    <Icon>{googleIcon}</Icon>*/}
+          {/*    haileymiller298@gmail.com*/}
+          {/*  </Row>*/}
+          {/*</Section>*/}
+          <Section>
+            <Row
+              onClick={() => {
+                goToAccountPage();
+              }}
+            >
+              <Icon>{gearIcon}</Icon>
+              Manage account
+            </Row>
+          </Section>
+          <Section>
+            <Row
+              onClick={() => {
+                logout();
+                setIsOpen(false);
+              }}
+            >
+              <Icon>{logoutIcon}</Icon>
+              Sign out
+            </Row>
+          </Section>
+        </Content>
+      )}
+    </DropDownDiv>
+  );
+};
+const Name = ({ children }) => <div style={primaryIdStyle}>{children}</div>;
+const primaryIdStyle: React.CSSProperties = {
+  fontWeight: 600,
+  fontSize: '16px',
+  lineHeight: '20px',
+  textAlign: 'left',
+};
+
+const Icon = (
+  props_: React.DetailedHTMLProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    HTMLDivElement
+  >
+) => {
+  const { style, ...props } = props_;
+  return (
+    <div
+      style={{
+        marginRight: 10,
+        display: 'flex',
+        alignItems: 'center',
+        ...style,
+      }}
+      {...props}
+    />
+  );
+};
+const DropDownDiv = ({ children }) => (
+  <div
+    style={{
+      position: 'relative',
+      color: '#374151',
+    }}
+  >
+    {children}
+  </div>
+);
+const Content = ({ children }) => (
+  <div
+    style={{
+      position: 'absolute',
+      right: 0,
+      background: 'white',
+      width: 246,
```

Comment:
Let's use `min-width` in case there is more content.

Can you test overflowing content as well and make sure it works as you expect, please?

Reply:
changed to min-width
![Screen Shot 2022-09-28 at 12 22 45](https://user-images.githubusercontent.com/24594168/192870726-3f0d832f-782c-4049-adf2-ebea600810ea.png)
additional content causes vertical growth, which I like